src/comgr.cpp

```diff
@@ -106,9 +106,9 @@ MIOPEN_DECLARE_ENV_VAR(MIOPEN_DEBUG_SRAM_EDC_DISABLED)
 #endif
 #endif
 
-// Temporary workaround for SWDEV-308265.
-// '__hipGetPCH' is not available since 4.4
-#if HIP_SUPPORTS_PCH && (HIP_PACKAGE_VERSION_FLAT > 4004000000)
+// '__hipGetPCH' is not available in [4.4, 5.0). See SWDEV-308265.
+#if HIP_SUPPORTS_PCH && (HIP_PACKAGE_VERSION_FLAT >= 4004000000) && \
+    (HIP_PACKAGE_VERSION_FLAT < 5000000000)
```

Comment:
@junliume 
> @sunway513 should we cherry-pick it into 4.5.x as @atamazov has mentioned above?

Simple cherry-picking won't work. We'll need to also change 
```
(HIP_PACKAGE_VERSION_FLAT < 5000000000)
```
to
```
(HIP_PACKAGE_VERSION_FLAT < 4005<patch>)
```
where `<patch>` should be taken from HIP version of 4.5.x that contains fixed ROCClr.